fullscreen.bs

```diff
@@ -171,12 +174,17 @@ partial interface Document {
  <dd><p>Returns true if <var>document</var> has the ability to display <a>elements</a> fullscreen
  and <a>fullscreen is supported</a>, or false otherwise.
 
- <dt><code><var>document</var> . {{Document/fullscreenElement}}</code>
- <dd><p>Returns <var>document</var>'s <a>fullscreen element</a>.
-
  <dt><code><var>promise</var> = <var>document</var> . {{Document/exitFullscreen()}}</code>
  <dd><p>Stops <var>document</var>'s <a>fullscreen element</a> from being displayed fullscreen and
  fulfills <var>promise</var> when done.
+
+ <dt><code><var>document</var> . {{DocumentOrShadowRoot/fullscreenElement}}</code>
+ <dd><p>Returns the result of retargeting <var>document</var>'s <a>fullscreen element</a> against
+ <var>document</var>.
```

Comment:
Yeah, as this is in a note, if that looks good to you, it can just say "return xxx's fullscreen element".
Done.